src/mrinufft/operators/autodiff.py

```diff
@@ -20,14 +20,15 @@ class _NUFFT_OP(torch.autograd.Function):
     @staticmethod
     def forward(ctx, x, traj, nufft_op):
         """Forward image -> k-space."""
-        ctx.save_for_backward(x, traj)
+        ctx.save_for_backward(x)
```

Comment:
@AsmaTANABEN just to let you know that I felt we dont need to savce the trajectory, so I removed that.